finetune/finetune.yaml

```diff
@@ -1,6 +1,7 @@
 General:
   base_model: EleutherAI/gpt-j-6b
   gpt_base_model: true
+  log_step: 10
```

Comment:
- This change is irrelevant to the purpose of this PR. Can you submit a sperate PR so we make each PR clear and have minimal changes? 
- Don't forget to update doc for the new parameter.
- It seems better to put this in "Training" section. I plan to add another parameter `report_to` that people can configure for example tensorboard. These two parameters will work together.

Reply:
removed